maqs-selenium/src/test/java/com/cognizantsoftvision/maqs/selenium/SeleniumConfigUnitTest.java

```diff
@@ -17,7 +16,7 @@
 /**
  * Selenium configuration tests.
  */
-public class SeleniumConfigUnitTest extends BaseGenericTest {
+public class SeleniumConfigUnitTest extends BaseSeleniumTest {
```

Comment:
Why add a specific base when generic is all we need?

Reply:
reverted back to baseGeneric